README.rst

```diff
@@ -65,46 +64,61 @@ To get structured info from a publication reference:
         'title': u'J. Phys.',
         'type': 'JOURNAL',
         'volume': u'A39',
-        'year': ''
-     }
+        'year': '',
+    }
 
-
-To extract references from a publication full-text PDF:
+To extract references from a PDF:
 
 .. code-block:: python
 
-    from refextract import extract_references_from_file
-    reference = extract_references_from_file("some/fulltext/1503.07589v1.pdf")
-    print(reference)
-    [
-            {'author': [u'F. Englert and R. Brout'],
-             'doi': [u'10.1103/PhysRevLett.13.321'],
-             'journal_page': [u'321'],
-             'journal_reference': ['Phys.Rev.Lett.,13,1964'],
-             'journal_title': [u'Phys.Rev.Lett.'],
-             'journal_volume': [u'13'],
-             'journal_year': [u'1964'],
-             'linemarker': [u'1'],
-             'title': [u'Broken symmetry and the mass of gauge vector mesons'],
-             'year': [u'1964']}, ...
-    ]
-
-You can also extract directly from a URL:
+    >>> from refextract import extract_references_from_file
+    >>> references = extract_references_from_file('1503.07589.pdf')
+    >>> print(references[0])
+    {
+        'author': [u'F. Englert and R. Brout'],
+        'doi': [u'doi:10.1103/PhysRevLett.13.321'],
+        'journal_page': [u'321'],
+        'journal_reference': [u'Phys. Rev. Lett. 13 (1964) 321'],
+        'journal_title': [u'Phys. Rev. Lett.'],
+        'journal_volume': [u'13'],
+        'journal_year': [u'1964'],
+        'linemarker': [u'1'],
+        'raw_ref': [u'[1] F. Englert and R. Brout, \u201cBroken symmetry and the mass of gauge vector mesons\u201d, Phys. Rev. Lett. 13 (1964) 321, doi:10.1103/PhysRevLett.13.321.'],
```

Comment:
I wondered a bit about the unicode code points `\u201c` and `\u201d` in the extraction example. For the `rawref` string that's ok. For actual extracted values it begs the question whether there are certain equivalence classes, for example various types of hyphens. That's an actual issue for the underlying code, not the docs under review here.